packages/flutter/test/material/tooltip_test.dart

```diff
@@ -725,38 +722,69 @@ void main() {
     ));
   });
 
-  testWidgets('Tooltip default size, shape, and color test for Desktop', (WidgetTester tester) async {
+  testWidgets('Does tooltip ends up with the right default height, shape, and color on Desktop', (WidgetTester tester) async {
     // Regressing test for https://github.com/flutter/flutter/issues/68601
     final GlobalKey<TooltipState> tooltipKey = GlobalKey<TooltipState>();
     await tester.pumpWidget(
       MaterialApp(
         home: Tooltip(
           key: tooltipKey,
           message: tooltipText,
-          child: const SizedBox(
-            width: 0.0,
-            height: 0.0,
-          ),
+          child: const SizedBox.shrink(),
         ),
       ),
     );
     tooltipKey.currentState?.ensureTooltipVisible();
-    await tester.pump(const Duration(seconds: 2)); // faded in, show timer started (and at 0.0)
+    await tester.pump(const Duration(seconds: 2));
 
-    final RenderParagraph tooltipRenderParagraph = tester.renderObject<RenderParagraph>(find.text(tooltipText));
-    expect(tooltipRenderParagraph.textSize.height, equals(10.0));
```

Comment:
I think we can do a switch case on defaultTargetPlatform and do different expect value instead of removing the check. But this may not be needed if macos also uses 12px